md/tomcat-bundle.md

```diff
@@ -106,30 +106,38 @@ For production, you are recommended to use one of the supported databases, with
-    1. Comment the default embedded h2 database configuration (preceding the lines with a #)
-    2. Uncomment the configuration for your database vendor (PostgreSQL, Oracle, SQL Server, or MySQL)
-    3. Change the default values for your database configuration to point to an existing database instance and valid credentials
-  Warning: this must be done for 2 different datasources in the file: **resource.ds1.** (for engine and configuration data) and **resource.ds2.** (for BDM data, optional but handy to configure in case some day you need to use it)
-3. Edit file `[TOMCAT_HOME]`/conf/Catalina/localhost/ **bonita.xml**
-    1. Comment the default embedded H2 database configuration (with `<!--` and `-->` around the lines to comment)
-    2. Uncomment the configuration for your database vendor (PostgreSQL, Oracle, SQL Server, or MySQL)
-    3. Change the default values for your database configuration to point to an existing database instance and valid credentials
-  Warning: this must be done for 2 different datasources in the file: **bonitaSequenceManagerDS** (for engine and configuration data, same base as **resource.ds1.**) and **NotManagedBizDataDS** (for BDM data, same base as **resource.ds2.**)
-4. Edit file sentenv.sh (Unix system) or setenv.bat (Windows system)
-    1. For engine and configuration data, change the **DB_OPTS** property and change the default **h2** value for the one corresponding to your database vendor 
-    2. For BDM data, change the **BDM_DB_OPTS** property and change the default **h2** value for the one corresponding to your database vendor 
 
+::: info
+What is the **bonita-start.sh** script doing?
+
+The **bonita-start** script does the following:
+
+1. Runs the **`setup init`** command:
+    1. initializes the Bonita BPM internal database (the one you defined in file `setup/database.properties`): creates the tables that Bonita BPM uses internally + stores the configuration in database.
+    2. install the license files (Subscription editions only) in the database.
+2. Runs the **`setup configure`** command:
+    The Setup Configure command auto-configures the Tomcat environment to access the right databases:
+    1. It updates the files sentenv.sh (Unix system) and setenv.bat (Windows system) to set the database vendor values for **Bonita BPM internal database** & **Business Data database**
+    2. It updates the file `[TOMCAT_HOME]`/conf/ **bitronix-resources.properties** with the values you set in file `database.properties`
+    3. It updates the file `[TOMCAT_HOME]`/conf/Catalina/localhost/ **bonita.xml** with the values you set in file `database.properties`
+    4. It copies your database vendor-specific drivers from `[TOMCAT_HOME]/setup/lib` to `[TOMCAT_HOME]/lib/bonita`
+3. Starts the Tomcat bundle
+
+Advanced users: if you need to finely tune your Tomcat bundle, see [Bundle auto-configuration](BonitaBPM_platform_setup.md#run_bundle_configure) for advanced usage of the configure command):
```

Comment:
change wording using imperative like : 
Check out Bundle aut-configuration to finely tune your tomcat bundle for advanced usage, bla bla bla